docs/containerd-env-setup.md

```diff
@@ -124,18 +126,28 @@ sudo systemctl restart containerd
 ```
 
 ### Using Runtime-level Snapshotter in Container for Nydus
-Containerd (>= v1.7.0) supports configuring the ```runtime-level``` snapshotter. By following the steps below, we can declare runtimes that use different snapshotters:
-#### Step 1: Configure Containerd
+
+Containerd (>= v1.7.0) supports configuring the `runtime-level` snapshotter. By following the steps below, we can declare runtimes that use different snapshotters:
+
+#### Step 1: Apply Containerd Patches
+
+1. [Patch 1](https://github.com/nydusaccelerator/containerd/commit/0959cdb0b190e35c058a0e5bc2e256e59b95b584): fixes the handle of sandbox run and container create for runtime-level snapshotter;
+2. [Patch 2](https://github.com/nydusaccelerator/containerd/commit/b98e71e858a213419c95b79fa0dcfd782ec7a785): always check the content store and fetch the missing blob before the snapshot unpack.
```

Comment:
Are these patches sent to containerd for review?

Reply:
Haven't yet, it is just a quick workaround, it seems we need more investigation for the complete fixup, patch 2 is [here](https://github.com/containerd/containerd/pull/8878).

Reply:
ok...